Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+knowledge, education, commerce and shopping, social experiences, civic functions, entertainment, and more.
+
+The Web's amazing success has also led to many unintended consequences that harm society: 
+openness and anonymity have given rise to scams, phishing, and fraud. 
+The ease of gathering personal information has led to business models that mine and sell detailed user data, without people's awareness or consent. 
+Rapid global information sharing has allowed misinformation to flourish and be exploited for political and commercial gain. 
+Technology is not neutral; 
+new technologies enable new actions and new possibilities, and we must take responsibility to address the impacts of the Web.
+
+We believe the World Wide Web should be inclusive and respectful of its users: 
+a Web that supports truth over falsehood, people over profits, humanity over hate. 
+The W3C’s Technical Architecture Group's work to clearly define [Ethical Web Principles](https://www.w3.org/2001/tag/doc/ethical-web-principles/) 
+is a strong basis to improve the ethical integrity of the Web. 
+The Web has had a tremendous impact on the world, and will continue to grow its impact in the future, 
+by expanding reach, knowledge, education and services even more broadly.
+
+### The W3C
+
+The World Wide Web Consortium (W3C) was founded as an organization to provide a consistent architecture across the rapid pace of progress in the Web, 
+and to build a common community to support its development. 
+It has become an association where diverse voices from around the world and industries work together to evolve the Web.
+
+To build a better future, the W3C must rise even further to the challenge of improving the Web's fundamental integrity, 
+while continuing to expand the Web’s scope and reach. 
+We must embody in our work the core values of the Web itself, 
+by supporting the Ethical Web Principles with Operational Principles of the W3C.
+
+The W3C’s impact will be clearly demonstrated by how it leads the Web forward: 
+by being inclusive, principled, and continually striving to make the Web better through these principles and the Ethical Web Principles. 
+As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+## Operational Principles of the W3C
+
+* The W3C intentionally involves stakeholders from end to end in building the Web: developers, content creators, and end users.
+  Our work will not be dominated by any person, company, or interest group, but uses consensus from the community
+  to put the needs of users first: above authors, publishers, and implementers.
+* We believe in the diversity of the Web, and its inclusion of participants
+  from different geographical locations, cultures, languages, disabilities, gender identities, industries, user groups, organizational sizes, and more.
+  In addition to ensuring the W3C serves the needs of the entire Web user base,
```

Comment:
I'm not sure that this is "in addition" - I think it's critical we broaden participation if we want to serve the entire Web user base.

Reply:
Reworded in #111.